website/en/docs/types/utilities.md

```diff
@@ -578,6 +578,23 @@ function makeParamStore<T>(storeClass: Class<ParamStore<T>>, data: T): ParamStor
 (makeParamStore(ParamStore, 1): ParamStore<boolean>); // failed because of the second parameter
 ```
 
+## `Shape<T>` <a class="toc" id="toc-shape" href="#toc-shape"></a>
+
+Copies the shape of the type supplied, but marks every field optional.
```

Comment:
Maybe not use "Shape" in the definition of "Shape"? How does "Copies an existing type and makes each of its properties optional" sound?